vignettes/SpatialOmicsOverlay.Rmd

```diff
@@ -33,10 +33,9 @@ Download package tarball from NanoString's GeoScriptHub (link pending)
 ```{r, eval=FALSE}
 if (!requireNamespace("BiocManager", quietly=TRUE))
     install.packages("BiocManager")
-BiocManager::install("GeomxTools")
 
-install.packages("path/to/SpatialOmicsOverlay_0.99.11.tar.gz", 
-                 dependencies = TRUE, repos = NULL)
+BiocManager::install("GeomxTools")
+BiocManager::install("SpatialOmicsOverlay")
```

Comment:
That code chunk does not get run `{r, eval=FALSE}` in the building of the vignette so it might error on the build in a different way than you are suggesting. I will be sure to be on top of this with the reviewer